src/Item/Item.service.ts

```diff
@@ -637,13 +645,15 @@ export class ItemService {
       : null
 
     const upsertedItem: ItemAttributes = await Item.upsert(attributes)
+    let curation: ItemCurationAttributes | undefined
     if (dbItem && attributes.local_content_hash) {
       // Update the Item Curation content_hash
-      await ItemCuration.update(
-        { content_hash: attributes.local_content_hash },
-        { item_id: attributes.id, status: CurationStatus.PENDING }
+      curation = await ItemCuration.updateByItemIdAndStatus(
+        attributes.id,
+        CurationStatus.PENDING,
+        { content_hash: attributes.local_content_hash }
       )
     }
-    return Bridge.toFullItem(upsertedItem, dbCollection)
+    return { item: Bridge.toFullItem(upsertedItem, dbCollection), curation }
```

Comment:
This is a breaking change in the API. I think that we should request the curation whenever the item is updated in the UI.

Reply:
Yeah, that's the right call. I've updated https://github.com/decentraland/builder/pull/1968 to fetch the item curation again, so I'm closing this PR.